app/src/main/java/zechs/drive/stream/data/model/DriveFile.kt

```diff
@@ -21,5 +21,9 @@ data class DriveFile(
 
     val isShortcut = mimeType == "application/vnd.google-apps.shortcut"
 
+    val isShortcutFolder = shortcutDetails.targetMimeType == "application/vnd.google-apps.folder"
+
+    val isShortcutVideo = shortcutDetails.targetMimeType?.startsWith("video/")
```

Comment:
Pretty sure this would return `Boolean?`. It can be avoided by adding a default `false` using the [elvis operator](https://kotlinlang.org/docs/null-safety.html#elvis-operator)